test/podman_import.bats

```diff
@@ -13,133 +13,85 @@ function setup() {
 }
 
 @test "podman import with source and reference" {
-    skip "Test needs to be converted to podman run bash -c"
-    start_crio
-    run bash -c crioctl pod run bash -c --config "$TESTDATA"/sandbox_config.json
+    run bash -c "${PODMAN_BINARY} ${PODMAN_OPTIONS} pull $IMAGE"
```

Comment:
this image is cached, so if you add the following to the head:
 
```
function setup() {
    copy_images
}
```

there is no need to pull the image

Reply:
Hi @baude 

The copy_images function is not working for podman. I tried to pull image manually with copyimg provided in under test. But seems the image it prepared has some format differences with current podman(compare the image dir with diff from podman and copyimg), so the podman can not recognized the image.  This is a work around for the copyimg problem. We can remove it after the problem is fixed.

Reply:
@ypu you do not pull anything ... the copyimg seeds the local image storage.  you can simply then run the image.  try changing $IMAGE to $ALPINE

Reply:
Hi @baude ,

Checked the CI tasks, seems the copyimg there is working. But it is not working in my test env. This case may not be a good example but I tried podman_inspect. It always failed with this error:
 ✗ podman inspect image
   (in test file test/podman_inspect.bats, line 16)
     `[ "$status" -eq 0 ]' failed
   error getting image "docker.io/library/alpine:latest": unable to locate image "docker.io/library/alpine:latest": image not known
   No JSON object could be decoded

For this test case, the command podman run will always pull image by itself in my machine. Do I need some special setup for copyimg working? This influence a lot of tests.

Reply:
And here is the error message if I remove the pull image step from my machine. You can see that the podman run command is pull image by itself:
1..4
not ok 1 podman import with source and reference
# (in test file test/podman_import.bats, line 21)
#   `[ "$status" -eq 0 ]' failed
# Trying to pull docker.io/library/alpine:latest...
# Getting image source signatures
# Copying blob sha256:ff3a5c916c92643ff77519ffa742d3ec61b7f591b6b7504599d95a4a41134e28
 1.97 MB / 1.97 MB [========================================================] 0s
# Copying config sha256:3fd9065eaf02feaf94d68376da52541925650b81698c53c6824d92ff63f98353
 1.48 KB / 1.48 KB [========================================================] 0s
# Writing manifest to image destination
# Storing signatures
# cb3359dfe108c3570f41dc2b099b7337040150a4189fe4bf7092e528f74b6f43
# too many arguments given, need 1 at most.
# bash: line 1: Getting: command not found
# bash: line 2: Copying: command not found
# bash: line 3: $'\r': command not found
# bash: line 4: Copying: command not found
# bash: line 5: $'\r': command not found
# bash: line 6: Writing: command not found
# bash: line 7: Storing: command not found
# bash: line 8: cb3359dfe108c3570f41dc2b099b7337040150a4189fe4bf7092e528f74b6f43: command not found
# cb3359dfe108c3570f41dc2b099b7337040150a4189fe4bf7092e528f74b6f43

Reply:
did you use $ALPINE instead of $IMAGE ?

Reply:
In the podman run command I always use $ALPINE to keep it the same with other test. So after remove the podman pull command, there is no more $IMAGE in the script. And I didn't not change anythign in the other tests(for example podman_inspect.bats). So this is not caused by the wrong variable name. I am not sure if this is caused by different env. Now I am using RHEL7 for test, will try it in Fedora.

Reply:
Hi @baude 

Tested it with Fedora 27 and copyimg works well. Not sure which lib or compiler caused this problem.
go version for f27: golang-1.9.2-1.fc27.x86_64
go version for rhel7: golang-1.9.2-4.el7.x86_64

Do I need check any other rpms version?